src/CirquePinnacle.h

```diff
@@ -804,25 +794,29 @@ class PinnacleTouchSPI : public PinnacleTouch
      * @param slaveSelectPin The "slave select" pin output to the Pinnacle
      * ASIC.
      */
-    PinnacleTouchSPI(uint16_t dataReadyPin, uint8_t slaveSelectPin);
+    PinnacleTouchSPI(pinnacle_gpio_t dataReadyPin, pinnacle_gpio_t slaveSelectPin);
     /**
      * Starts the driver interface on the appropriate SPI bus.
-     * @rst
-     * :Return:
-     *     - ``true`` if the Pinnacle ASIC was setup and configured properly (with data
-     *       feed enabled using Relative mode).
-     *     - ``false`` if the Pinnacle ASIC was unresponsive for some reason (all further
-     *       operations will be nullified by setting `Data Mode`_ to ``0xFF``).
-     * @endrst
+     * @returns The same value as `PinnacleTouch::begin()`.
      */
     bool begin();
+    /**
+     * Starts the driver interface on the specified SPI bus object. This function is useful if the
+     * application uses more than one SPI bus.
+     * 
+     * @param spi_bus A reference to the instantiated object that corresponds to
+     * a specific SPI bus.
+     * @returns The same value as `PinnacleTouch::begin()`.
+     */
+    bool begin(_SPI* spi_bus);
 
 private:
     void rapWrite(uint8_t, uint8_t);
     void rapWriteBytes(uint8_t, uint8_t*, uint8_t);
     void rapRead(uint8_t, uint8_t*);
     void rapReadBytes(uint8_t, uint8_t*, uint8_t);
     uint8_t _slaveSelect;
+    _SPI* spi;
```

Comment:
This must be where the increase in memory is from... Seems unavoidable since I want to support the possibility of multiple SPI/I2C busses.